hips/hip-9999.yaml

```diff
@@ -0,0 +1,58 @@
+---
+hip: 9999
+title: "Support for multiple yaml documents in stdin/file"
+authors: [ "Jan-Otto Kröpke <mail@jkroepke.de>" ]
+created: "2020-08-13"
+type: "feature"
+status: "draft"
+---
+
+## Abstract
+
+Helm allows values through stdin. But if stdin contains a multi yaml document, only the first document will be parsed. Currently pass value files through stdin is not really a good replacement if there are multiple value files.
+
+## Motivation
+
+There are servial projects to handle secrets inside helm values. For example https://github.com/jkroepke/helm-secrets. helm-secrets acts as wrapper script. The helm-secrets plugin will be parsed all value files, decrypt them if needed an internally re-call helm with the same parameters and modified value files. For a short time period, unencrypted values are on disk and can be leaked. If helm-secrets could pass the values through stdin, such unencrypted values are not longer leakable. See also [ArgoCD security advisory]. 
+
+## Rationale
+
+Helm already has a specification to merge YAML value files. The existing merge strategy should be re-used to merge multiple yaml documents. There is be no different between merge 2 files or 2 documents in 1 file or 2 documents through stdin.
+
+## Specification
+
+Helm already has a specification to merge YAML value files. The existing merge strategy should be re-used to merge multiple yaml documents. There is be no different between merge 2 files or 2 documents in 1 file or 2 documents through stdin.
```

Comment:
This is the exact same content you put under the "Rationale". The "Specification" should go into greater detail as to how the feature will work.

For example, providing a link to the "specification to merge YAML value files" would be helpful. If none exists, please describe the process. You should also take the time to describe the "existing merge strategy" in this section as well.